app/routes/schemas/polygon.py

```diff
@@ -36,26 +38,20 @@ def __init__(self, **args):
     @field_validator("bbox", mode="before")
     def validate_bbox(cls, v):
         if len(v) not in [4, 6]:
-            raise ValueError("Bounding box (bbox) must have 4 or 6 elements.")
+            raise_http_exception(422, "bbox_length", url="None")
```

Comment:
not sending the url is to lose information. If you let the error raise up to the exception_handlers it will not be lost